neurodocker/interfaces/interfaces.py

```diff
@@ -250,6 +259,20 @@ def __init__(self, *args, **kwargs):
         super().__init__(self._name, *args, **kwargs)
 
 
+class NDFreeze(_BaseInterface):
+    """Create instance of NDFreeze oject."""
+
+    _name = "ndfreeze"
+
+    def __init__(self, date, *args, **kwargs):
```

Comment:
note that later on we would also add options, at least one, to e.g. force downgrade of already installed packages to the ones available from the snapshots (the last TODO in https://github.com/neurodebian/neurodebian/pull/39 ;))